lib/MusicBrainz/Server/Data/Edit.pm

```diff
@@ -76,7 +77,7 @@ sub _new_from_row
         expires_time => $row->{expire_time},
         auto_edit => $row->{autoedit},
         status => $row->{status},
-        raw_data => $row->{data},
+        raw_data => decode('utf-8', $row->{data}),
```

Comment:
This one is different from `$data` value from above, right?

    my $data = JSON::XS->new->utf8->decode($row->{data});